guides/v2.2/frontend-dev-guide/layouts/layout-overview.md

```diff
@@ -57,19 +55,19 @@ For a particular page, its layout is defined by two major layout components: *pa
 
 Following are the definitions of each layout file type:
 
-* *Page layout*: an {% glossarytooltip 8c0645c5-aa6b-4a52-8266-5659a8b9d079 %}XML{% endglossarytooltip %} file declaring a page wireframe inside the `<body>` section of the HTML page markup, for example, two-column page layout. 
+* *Page layout*: an {% glossarytooltip 8c0645c5-aa6b-4a52-8266-5659a8b9d079 %}XML{% endglossarytooltip %} file declaring a page wireframe inside the `<body>` section of the HTML page markup, for example, two-column page layout.
 * *Page configuration*: an XML file declaring detailed structure, contents and meta-information of a page (includes the `<html>`, `<head>`, and `<body>` sections of the HTML page markup).
 * *Generic layout*: an XML file declaring page detailed structure and contents inside the `body` section of the HTML page {% glossarytooltip 8f407f13-4350-449b-9dc5-217dcf01bc42 %}markup{% endglossarytooltip %}. Used for pages returned by AJAX requests, emails, HTML snippets, and so on.
 
-For details, refer to <a href="{{ page.baseurl }}/frontend-dev-guide/layouts/layout-types.html" target="_blank">Layout file types</a>.
+For details, refer to [Layout file types]{:target="_blank"}.
```

Comment:
move all `{:target="_blank"}` to the Link definitions:
```
[reference]: link
{:target="_blank"}
[other_reference]: other/link
{:target="_blank"}
```

Reply:
Wasnt sure the best way to do this when multiple links share the same link but some are not target="_blank"

Reply:
All links pointing to other than this page resources should be `target="_blank"`